go.mod

```diff
@@ -130,7 +130,7 @@ require (
 )
 
 replace (
-	github.com/cosmos/cosmos-sdk => github.com/sei-protocol/sei-cosmos v0.1.67
+	github.com/cosmos/cosmos-sdk => ../sei-cosmos //github.com/sei-protocol/sei-cosmos v0.1.68
```

Comment:
self reminder to revert this prior to merging

Reply:
fyi, i bumped it to `v0.1.69`, should be in master